report/setup/preamble.tex

```diff
@@ -317,4 +317,51 @@
     morecomment=[s][\color{RedTypename}]{\#[}{]},
     stringstyle=\color{GreenString},
     string=[b]"
+}
+
+
+\definecolor{bluekeywords}{rgb}{0,0,1}
+\definecolor{greencomments}{rgb}{0,0.5,0}
+\definecolor{redstrings}{rgb}{0.64,0.08,0.08}
+\definecolor{xmlcomments}{rgb}{0.5,0.5,0.5}
+\definecolor{types}{rgb}{0.17,0.57,0.68}
+
+\lstset{language=cs,
```

Comment:
Why are you defining C# again? We already have `CSharp` as a language.

Reply:
I will yeet